### ISSUE TITLE
Package 'Mariuzzo.Extensions 13.3.6.32877' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.2'. This package may not be fully compatible with your project.

### ISSUE THREAD

**Issue by VictorioBerra** [rmariuzzo/Mariuzzo.Web.Mvc.Extras#1]
Severity	Code	Description	Project	File	Line	Suppression State
Warning	NU1701	Package 'Mariuzzo.Extensions 13.3.6.32877' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.2'. This package may not be fully compatible with your project.1	

Got a version which targets the netstandard2.2?

**Comment by rmariuzzo** [comment]
Unfortunately, I don't. Can a PR solve this?

**Comment by VictorioBerra** [comment]
Yes, you would convert your project to a modern core project and target both full framework and netstandard. Kind of like how these people did it https://github.com/messagebird/csharp-rest-api/pull/40
